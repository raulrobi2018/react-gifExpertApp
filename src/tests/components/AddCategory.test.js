const {shallow} = require("enzyme");
const {AddCategory} = require("../../components/AddCategory");

describe("Test en componente AddCategory", () => {
    const setCategories = () => {};

    const wrapper = shallow(<AddCategory category={setCategories} />);

    test("should return AddCategory correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
