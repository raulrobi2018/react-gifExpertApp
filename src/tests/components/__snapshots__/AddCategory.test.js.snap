// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en componente AddCategory should return AddCategory correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
